### ISSUE TITLE
geodDist(..., alongPath=FALSE) fails without longitude2 and latitude2

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1577]
See below
```R
> library(oce)
> geodDist(c(-63,-63),c(45,55),alongPath=TRUE)
[1]    0.000 1112.286
> geodDist(c(-63,-63),c(45,55),alongPath=FALSE)
Error in do_geoddist(longitude1, latitude1, longitude2, latitude2, a,  : 
  Not compatible with requested type: [type=NULL; target=double].
```
I want this to work for my ocean-glider QC shiny app, so I'll fix this bug today.

**Comment by dankelley** [comment]
The docs say
> Case 2: longitude1 is a vector. If longitude2 and latitude2 are not given, then the return value is a vector containing the distances of each point from the first one, or the distance along the path connecting the points, according to the value of alongPath. 

and this is not working.  The fix will be to make it be as if
```R
> geodDist(c(-63,-63),c(45,55),-63,45,alongPath=FALSE)
[1]    0.000 1112.286
```

**Comment by dankelley** [comment]
Fixed in 4a2e36866b69bcb8ca5a1f7b48700dbac1da064e, as shown below:
```R
> library(oce)
> geodDist(c(-63,-63),c(45,55),-63,45,alongPath=FALSE)
[1]    0.000 1112.286
> geodDist(c(-63,-63),c(45,55),alongPath=FALSE)
[1]    0.000 1112.286
```

**Comment by dankelley** [comment]
Oh, I forgot to close this.
